back/lib/tasks/single_use/20240214_migrate_file_upload_custom_fields.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# TODO: Can we do this in pure SQL by joining the file upload fields?
```

Comment:
Tricky because the ID is stored in values. Alternatively, it's possible to first fetch all IdeaFile ids with sth like `ids = ideas.flat_map { |idea| idea.custom_field_values.keys }` and then fetch `IdeaFile` records like `IdeaFile.where(id: ids).index_by(&:id)` to avoid n+1 queries for `IdeaFile`.

But I believe the current implementation is good enough.